core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1569,6 +1570,34 @@ class WorkspaceService(protected val userInfo: UserInfo,
     }
   }
 
+  def retrySubmission(workspaceName: WorkspaceName, submissionRequest: SubmissionRequest, submissionId: String): Future[SubmissionReport] = {
+    val workspaceContext = getWorkspaceContext(workspaceName)
+    val submissionStatus = getSubmissionStatus(workspaceName, submissionId)
+    submissionStatus.onComplete {
+      case Failure(_) => throw new RawlsExceptionWithErrorReport(errorReport = ErrorReport(StatusCodes.NotFound, s"submission with id ${submissionId} does not exist in ${workspaceName.namespace}/${workspaceName.name}"))
+      case Success(status) =>
+        val newSubmission = Submission(submissionId = submissionId.toString,
+          submissionDate = DateTime.now(),
+          submitter = WorkbenchEmail(userInfo.userEmail.value),
+          methodConfigurationNamespace = submissionRequest.methodConfigurationNamespace,
+          methodConfigurationName = submissionRequest.methodConfigurationName,
+          submissionEntity = submissionEntityOpt,
+          workflows = submissionRequest.retryType.filterWorkflow(status.workflows),
+          status = SubmissionStatuses.Submitted,
+          useCallCache = submissionRequest.useCallCache,
```

Comment:
For the MVP, we want to take all these values from the previous submission, not from the request. As a future change, we want to support the user optionally overriding the previous submission values in the request.

So currently, we just want to use the values from previous submission.